ext/fs/lib.rs

```diff
@@ -1488,7 +1493,12 @@ where
     debug!("op_realpath_async {}", path.display());
     // corresponds to the realpath on Unix and
     // CreateFile and GetFinalPathNameByHandle on Windows
-    let realpath = canonicalize_path(&path)?;
+    let realpath = canonicalize_path(&path).map_err(|error| {
+      default_err_mapper(
+        error,
+        format!("op_realpath_async '{}', Deno.realPath", path.display()),
```

Comment:
**issue:** Other `default_err_mapper` calls do not include the `Deno.realPath` method name in the formatted error message. I think it wold be proper here to align with the others and remove the extra JS side method name mention.

Reply:
I can change this up sometime today.